go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/gruntwork-io/terratest v0.41.3
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
-	github.com/lacework/go-sdk v1.1.3-0.20221214173411-1ec57f679902
+	github.com/lacework/go-sdk v1.9.2-0.20230207104645-f46a8c8ec140
```

Comment:
should we update go-sdk dep to latest `v1.12.0`

Reply:
Updated to use the latest SDK